experimental/federated_plugin/federated_plugin_macos/macos/Classes/FederatedPluginMacosPlugin.swift

```diff
@@ -0,0 +1,33 @@
+import Cocoa
```

Comment:
nit: Copyright header needed.

Reply:
Merged :D

Added copyright header for iOS too.